Team/teammembers/views.py

```diff
@@ -0,0 +1,37 @@
+from django.http import JsonResponse, HttpResponse
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from .serializers import TeamMemberSerializer
+from django.shortcuts import get_object_or_404
+
+import json
+
+from .models import TeamMember
+
+
+class TeamMemberListView(APIView):
+	def get(self, request):
+		team_member_list = TeamMember.objects.all()
+		serializer = TeamMemberSerializer(team_member_list, many=True)
+		return JsonResponse({'team_members': serializer.data})
+
+	def post(self, request):
```

Comment:
so you can create a new team member via post, put on list view and a create method on the the detail view. Why?